### PR TITLE
Add page about research software directory

### PR DESCRIPTION
**Pull request by rwood-97**
## Summary

Adds page about RSD - what is it and how to add stuff.

## Related issues

Relates to https://github.com/alan-turing-institute/hut23-open-source-sa/issues/19

### CODE REVIEW

**Review by edwardchalstrey1 — APPROVED** [review]
Reads well to me 🏆

File: content/docs/how_we_work/research-software-directory.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+To request access, you should send an email from your Turing email account to `rsd@esciencecenter.nl` stating the following information:
+
+- Your name.
+- Your affiliation(s) - i.e. The Alan Turing Institute
+- Your ORCID.
+- Any example links illustrating the content you wish to add.
+
+### How to join The Alan Turing Institute organisation on the RSD
+
+Once you have logged in to the RSD, you will need to send [one of the RSD admins](https://github.com/alan-turing-institute/research-engineering-group/wiki/The-REGistry) a message asking to be added to the Alan Turing Institute organisation.
```
Comment (yongrenjie) [inline content/docs/how_we_work/research-software-directory.md:37]:
```suggestion
Once you have logged in to the RSD, you will need to send [one of the RSD admins](https://github.com/alan-turing-institute/research-engineering-group/wiki/The-REGistry) a message asking to be added to the The Alan Turing Institute organisation.
```

**Review by yongrenjie — COMMENTED** [review]
The

File: content/docs/how_we_work/research-software-directory.md (hunk 1 above)
Comment (yongrenjie) [reply content/docs/how_we_work/research-software-directory.md:37]: Maybe we should add a test to CI to make sure that the 'The' is always capitalised?!

Comment (andrewphilipsmith) [reply content/docs/how_we_work/research-software-directory.md:37]: Yes to capitalising the 'T', but we don't need 'the The'.

Comment (rwood-97) [reply content/docs/how_we_work/research-software-directory.md]: will just fix this

**Comment by rwood-97** [comment]
@andrewphilipsmith am I safe to ignore the CI failing and merge this?

**Comment by andrewphilipsmith** [comment]
> @andrewphilipsmith am I safe to ignore the CI failing and merge this?

@dsj976 or @JimMadge are best placed to answer this.

However, given that it looks like the same issue mentioned here: https://github.com/alan-turing-institute/REG-handbook/issues/149, I would guess that it would be OK to merge it in its current state.

**Comment by dsj976** [comment]
Yes, if that's the only failure. However, I think the GitHub Pages won't be updated as it won't be able to run Hugo. I will try and fix this today or tomorrow at some point

**Comment by penelopeysm** [comment]
@all-contributors add @rwood-97 for content, add @edwardchalstrey1 for review

**Comment by penelopeysm** [comment]
@all-contributors
please add @rwood-97 for content.
please add @edwardchalstrey1 for review.

**Comment by penelopeysm** [comment]
@all-contributors add @rwood-97 for content

**Pull request merged**